### PR TITLE
[EXP-770] Atualizar fluxo de antecipação para passar senha na rota 

### PR DESCRIPTION
**Pull request by diegodsgarcia**
## Contexto
O objetivo desse PR é remover a rota de verify antes de enviar o request de criar antecipação,  e na rota de criar antecipação, enviar a senha no corpo da request.

## Checklist
- [x] Remover request de session verify
- [x] Adicionar password ao enviar request de criar antecipação
- [x] Tratar erro de password invalido na request de criar antecipação 

## Como testar?
- Rodar a pilot nesta branch
- Ir na página de antecipação
- Testar antecipação com a senha incorreta para ver se o aviso aparece corretamente
- Testar antecipação com a senha correta para ver se o saque é efetivado.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b7d67620a07f0a5324d4bdaa6c77ae77a39e0474
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b7d67620a07f0a5324d4bdaa6c77ae77a39e0474
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b7d67620a07f0a5324d4bdaa6c77ae77a39e0474
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b7d67620a07f0a5324d4bdaa6c77ae77a39e0474
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Review by Hrfreire — APPROVED** [review]

File: packages/pilot/src/pages/Anticipation/Anticipation.js [hunk 1]
```diff
@@ -777,13 +745,27 @@ class Anticipation extends Component {
           statusMessage: t('pages.anticipation.anticipation_success'),
           stepsStatus: getStepsStatus('result', 'success'),
         })
+
+        return this.calculateLimits()
+      })
+      .catch((error) => {
```
Comment (LucasRichter) [inline packages/pilot/src/pages/Anticipation/Anticipation.js:751]: Acho que vale trocar tudo por async/await

**Review by LucasRichter — APPROVED** [review]

**Pull request merged**